### ISSUE TITLE
Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by chuckcarpenter** [shipshapecode/website-astro#17]
Axe tool and Lighthouse state that contract is a serious issue in the nav tab for Contact due to color contrast. Also, body text on the light background is failing as too light for contrast.

**Comment by hannakim91** [comment]
Yeah areas that are not meeting contrast requirements pass for Large for the 

![Screen Shot 2022-10-11 at 3 18 07 PM](https://user-images.githubusercontent.com/62962173/195208459-556ed864-b43f-44ad-9917-c8e7b3911bae.png)
![Screen Shot 2022-10-11 at 3 19 38 PM](https://user-images.githubusercontent.com/62962173/195208943-03ae1783-c54f-4c7b-9f98-196811a2e79a.png)
![Screen Shot 2022-10-11 at 3 21 10 PM](https://user-images.githubusercontent.com/62962173/195209080-47fe8aa0-7e78-4150-b6b1-74255280af28.png)
